src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -41,6 +46,24 @@ void CryptoNative_EnsureLibSslInitialized()
 #elif OPENSSL_VERSION_NUMBER < OPENSSL_VERSION_1_1_0_RTM
     EnsureLibSsl10Initialized();
 #endif
+
+    // Get length of session master key and generate one.
+    SSL_CTX* ctx = SSL_CTX_new(TLS_method());
+    if (ctx != NULL)
+    {
+        s_sessionTicketsMasterKeyLength = SSL_CTX_ctrl(ctx, SSL_CTRL_SET_TLSEXT_TICKET_KEYS, 0, NULL);
+        if (s_sessionTicketsMasterKeyLength > 0)
+        {
+            s_sessionTicketsMasterKey = malloc((size_t)s_sessionTicketsMasterKeyLength);
```

Comment:
Nit: In general `calloc(elementCount, sizeof(elementType))` is preferred because it zero-inits.

Reply:
I can do that to match style. But the memory is initialized immediately after with RAND_bytes() or it will be freed if that fails. So there does not seems to be any advantage of zero-inits or initial zero states.